src/components/RestaurantItem.js

```diff
@@ -0,0 +1,28 @@
+import { IMAGE_SRC_BY_RESTAURANTS_CATEGORY } from "../constants/constants";
+
+const createRestaurantItem = ({
+  category,
+  name,
+  distance,
+  description,
+  imgSrc,
+}) => {
+  const li = document.createElement("li");
+  li.classList.add("restaurant");
+
+  const mappedImage = IMAGE_SRC_BY_RESTAURANTS_CATEGORY[category];
+
+  li.innerHTML = `
+      <div class="restaurant__category">
+        <img src="${mappedImage}" alt="${category}" class="category-icon" />
+      </div>
+      <div class="restaurant__info">
+        <h3 class="restaurant__name text-subtitle">${name}</h3>
+        <span class="restaurant__distance text-body">${distance}</span>
+        <p class="restaurant__description text-body">${description}</p>
```

Comment:
description이 옵셔널 값인 것을 감안해서 아래와 같이 조건부 렌더링 하는 것은 어떠신가요? description이 없을때 불필요한 p태그가 HTML에 남아있게 되어서요!

```suggestion
        ${description ? `<p class="restaurant__description text-body">${description}</p>` : ""}```

Reply:
제안주신 대로 조건부 렌더링이 나을 것 같아서 교체했습니다~!